intelmq/bots/parsers/shadowserver/config.py

```diff
@@ -85,10 +110,10 @@ def validate_ip(value):
         ('source.geolocation.cc', 'geo'),
         ('source.geolocation.region', 'region'),
         ('source.geolocation.city', 'city'),
-        # Other known fields  which will go into "extra"
+        # Other known fields which will go into "extra"
+        ('naics', int),
```

Comment:
> On 06 Jul 2016, at 17:44, Sebastian <notifications@github.com> wrote:
> 
> In intelmq/bots/parsers/shadowserver/config.py:
> 
> > @@ -85,10 +110,10 @@ def validate_ip(value):
> >          ('source.geolocation.cc', 'geo'),
> >          ('source.geolocation.region', 'region'),
> >          ('source.geolocation.city', 'city'),
> > -        # Other known fields  which will go into "extra"
> > +        # Other known fields which will go into "extra"
> > +        ('naics', int),
> 
> True. Is a NAICS or SIC of 0 valid? @aaronkaplan

Nope, I would not assume so.

http://www.census.gov/cgi-bin/sssd/naics/naicsrch?chart=2012
has the NAICS codes as XLS files and no, the value 0 is not defined there.

So, actually the converter must be able to either convert to an int or to the empty value (or ignore the field if it is empty in the first place).

a.


> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>